code/game/antagonist/outsider/ert.dm

```diff
@@ -39,10 +41,6 @@ var/datum/antagonist/ert/ert
 	player.equip_to_slot_or_del(new /obj/item/clothing/shoes/swat(src), slot_shoes)
 	player.equip_to_slot_or_del(new /obj/item/clothing/gloves/swat(src), slot_gloves)
 	player.equip_to_slot_or_del(new /obj/item/clothing/glasses/sunglasses(src), slot_glasses)
-	player.implant_loyalty(player) //aparantly they are loyalty implanted??
```

Comment:
Same reason we don't give to sec and the like, also, loyalty implants can get on the way of antag status, aka; being ert is classified as antag.